api/all/src/main/java/io/opentelemetry/api/trace/Tracer.java

```diff
@@ -22,7 +22,7 @@
  *
  * <pre>{@code
  * class MyClass {
- *   private static final Tracer tracer = OpenTelemetry.getTracer();
+ *   private static final Tracer tracer = OpenTelemetry.getTracer("instrumentation-library-name", "1.0.0");
```

Comment:
This is also not a currently existing API. We'd need to change this to either `GlobalOpenTelemetry` or (preferable) a lower-case `openTelemetry` instance.